### DISCUSSION TITLE
How to import all exports

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#5488] [Firefly III]
Hi All,

I have exported all options:
- accounts
- bills
- budgets
- categories
- piggies
- recurrences
- rules
- tags
- transactions

This because I want move from Postgres to MariaDB. (My MariaDB is automatic backup every night)

But how can I import all exports into firefly that now is connected to MariaDB?

Cheers,
Peter

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/exporting-data/

You may have a better shot migrating the database directly. Online you can find several scripts to migrate from postgres to mariadb. I'm not sure what the easiest way would be to import this data. I'm afraid Firefly III doesn't offer an easy way to import anything, except the transactions.
